build.assets/charts/Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM ubuntu:20.04
+# Stage to build the image, without FIPS entrypoint argument
+FROM ubuntu:20.04 AS teleport
+
+# Copy the deb archive
+ARG DEB_PATH
+COPY ${DEB_PATH?} /tmp/teleport.deb
```

Comment:
Why do we require a deb package here? Can't we download artifacts from .tar.gz for the appropriate platform that we build?

Reply:
Discussed offline, using package manager simplifies picking an appropriate artifact for the platform, otherwise we would have to reimplement this logic ourselves which could be especially challenging since our artifact tarballs aren't named consistently.